HelloWars/Arena/Games/TicTacToe/Models/Player.cs

```diff
@@ -1,23 +1,25 @@
-﻿using System;
+﻿using System.IO;
+using System.Net;
 using System.Windows;
-using Bot = BotClient.BotClient;
+using Newtonsoft.Json;
 
 namespace Arena.Games.TicTacToe.Models
 {
     public class Player
     {
-        private readonly Random _rand = new Random(DateTime.Now.Millisecond);
-
-        public Bot Bot;
+        public BotClient.BotClient Bot;
         public bool IsWinner;
         public BindableArray<Visibility> PlayerMovesArray; 
         public Point NextMove()
         {
-            return new Point
-            {
-                X = _rand.Next(0, 3),
-                Y = _rand.Next(0, 3),
-            };       
+            var webClient = new WebClient();
```

Comment:
Repleaced by new Helper which returns deserialized object of T type
  return WebClientHelper<Point>.GetResponseFromUrl(Bot.Url + "PerformNextMove");